cirq-core/cirq/contrib/svg/svg_test.py

```diff
@@ -57,3 +57,22 @@ def test_empty_moments():
     svg_2 = circuit_to_svg(cirq.Circuit(cirq.Moment()))
     assert '<svg' in svg_2
     assert '</svg>' in svg_2
+
+
+def test_gate_with_less_greater_str():
+    class CustomGate(cirq.Gate):
+        def _num_qubits_(self) -> int:
+            return 4
+
+        def _circuit_diagram_info_(self, _) -> cirq.CircuitDiagramInfo:
+            return cirq.CircuitDiagramInfo(wire_symbols=['<a', '<=b', '>c', '>=d'])
+
+    circuit = cirq.Circuit(CustomGate().on(*cirq.LineQubit.range(4)))
+    svg = circuit_to_svg(circuit)
+    import IPython.display
```

Comment:
I think we need to add ipython to the [requirements file](https://github.com/quantumlib/Cirq/blob/master/cirq-core/cirq/contrib/requirements.txt) and to mypy config.

Reply:
Since we are using iPython only in the tests and not in the library code itself; we need it add it to the dev requirements.   The dev requirements are defined in https://github.com/quantumlib/Cirq/blob/master/dev_tools/requirements/dev.env.txt which imports https://github.com/quantumlib/Cirq/blob/master/dev_tools/requirements/deps/dev-tools.txt which imports https://github.com/quantumlib/Cirq/blob/master/dev_tools/requirements/deps/notebook.txt which imports 
https://github.com/quantumlib/Cirq/blob/master/dev_tools/requirements/deps/ipython.txt

Therefore ipython already exists as part of the dev requirements. 


To make mypy happy, I've added a `# type: ignore` for this specific import.